File: hazi/kovacs_norbert/butorgyar/Butorlap.java
package butorgyar;

class Butorlap {
    private int hosszusag;
    private int szelesseg;
    private String tipus; 
    private static final int TAROLAP_AR = 5000;
    private static final int HATLAP_AR = 500;

    public Butorlap(String tipus, int hosszusag, int szelesseg) {
        this.tipus = tipus;
        this.hosszusag = hosszusag;
        this.szelesseg = szelesseg;
    }

    public int arSzamol() {
        int terulet = hosszusag * szelesseg;
        if ("tartolap".equals(tipus)) {
            return terulet * TAROLAP_AR;
        } else if ("hatlap".equals(tipus)) {
            return terulet * HATLAP_AR;
        } else 
            return 0;
    }}
    

File: python_kozepszint/Butorgyar/Butorlap.py
tartolap_ar  = 5000
hatlap_ar = 500

class Butorlap:
    """
    Ez egy bútorlap osztály, amit a butorokhoz adunk hozzá.
    parameter:
    tipus: tartolap vagy hatlap
    hosszusag: int
    szelesseg: int
    """
    def __init__(self,tipus,hosszusag,szelesseg):
        self.tipus = tipus
        self.hosszusag=hosszusag
        self.szelesseg=szelesseg
    
    def arSzamol(self):
        terulet = self.hosszusag*self.szelesseg
        if(self.tipus=='tartolap'):
            return terulet*hatlap_ar
        if(self.tipus=='hatlap'):
            return terulet*hatlap_ar
        else:
            return 0 